eureka2-client/src/main/java/com/netflix/eureka2/client/registration/RegistrationRequest.java

```diff
@@ -0,0 +1,42 @@
+package com.netflix.eureka2.client.registration;
+
+import rx.Observable;
+import rx.Subscriber;
+
+/**
+ * An observable that represent a registration request. The registration begins when the request is subscribed to,
+ * and ends when the request is unsubscribed from.
+ *
+ * The request observable also represent the success/error of the overall registration lifecycle and will onError
```

Comment:
What is a retry policy on this observable? As discussed we should support it, and so it would be good to document it here.